portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,31 +48,44 @@
 public class DataServlet extends HttpServlet {
   private static final Gson GSON = new Gson();
   private static final DatastoreService DATASTORE = DatastoreServiceFactory.getDatastoreService();
+  private static final BlobstoreService BLOBSTORE = BlobstoreServiceFactory.getBlobstoreService();
+  private static final ImagesService IMAGESERVICE = ImagesServiceFactory.getImagesService();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment");
+    Query query = new Query("Input");
```

Comment:
I would suggest make constants like ```Input``` to private static final. In such a way, we could catch some simple typo.